releasenotes/notes/agent-file-ownership-13f077681711995c.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The files provided by the agent and its configuration are no longer
+    owned by the `dd-agent` user but by `root` instead.
```

```suggestion
    owned by the ``dd-agent`` user but by ``root`` instead.
```

The release notes are rst, not md.